subsys/sal/common/sid_on_dev_cert/sid_on_dev_cert.c

```diff
@@ -93,7 +106,7 @@ static struct sid_on_dev_cert_context *context = NULL;
 static bool write_to_mfg_store(uint16_t value, const uint8_t *const buffer, uint16_t length)
 {
 	uint8_t *read_buf = NULL;
-	int32_t result = sid_pal_mfg_store_write(value, buffer, length);
+	int32_t result = tlv_write(&tlv_flash, value, buffer, length);
```

Comment:
Why to replace sid pal mfg api with tlv api in this module?
It requires addtional code, to operate on lower layer api